### PR TITLE
Basic installation instructions

### PR DESCRIPTION
**Pull request by noelwelsh**
Need to create a base project for the "some programming experience" group and link to it. Otherwise basic setup instructions are correct.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
 # SetUp
 Tools and steps to install them for the ScalaBridge course.
+
+## Software
+
+There are two routes to installing the software you need to develop Scala:
+
+- *If you already have tools you are comfortable with* then you can continue to use them. You will need the example project below and and [SBT][sbt]. If you go this route it is your responsibility to maintain your setup, though the mentors are likely to have experience with common tools like Emacs and Vim.
```
Comment (annabee) [inline README.md]: Two `and`s before SBT.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+
+- *If you haven't programmed before or don't have a strong preference* we recommend you install the [IntelliJ Scala Bundle][intellij-scala]. Scroll the page a bit until you see the "Downloads" heading, and download the appropriate package from there.
+
+## Base Project
+
+The base project provides some basic configuration for the work we'll do at ScalaBridge. Depending on your group you will have a different base project:
+
+- *If you don't have programming experience* get the [Creative Scala template][creative-scala-template]. Click the green "Clone or download" button and choose "Download ZIP". (If you know how to use Git you can clone or fork the repository instead.)
+
+- *If you have programming experience, but not with SCala* TBC
```
Comment (annabee) [inline README.md]: Scala

Comment (annabee) [reply README.md]: What were you thinking could go here?

Comment (noelwelsh) [reply README.md]: Thanks for the fixes. I've rolled them. For the base project for "some programming experience" I was thinking of a basic SBT Scala project will a very useful libraries added (e.g. Cats).

File: README.md (hunk 1 above)
Comment (jatcwang) [inline README.md]: Wondering whether we need to mention the need for JDK as well. Some students (even experienced) may not have it installed

Comment (annabee) [reply README.md]: I would just add the Oracle instructions and not fuff around with Open JDK. https://www.java.com/en/download/help/download_options.xml

**Pull request merged**